src/main/kotlin/Request.kt

```diff
@@ -0,0 +1,3 @@
+package request
+class Request(val httpMethod: String, val route: String) {
+}
```

Comment:
These symbols at the end of the line mean that you should have a blank line at the end of the file.

Reply:
Updated IDE settings to ensure every saved file ends with a line break